filter/expand_json.go

```diff
@@ -0,0 +1,133 @@
+package filter
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"sync/atomic"
+
+	"github.com/AdRoll/baker"
+	"github.com/jmespath/go-jmespath"
+)
+
+var ExpandJSONDesc = baker.FilterDesc{
+	Name:   "ExpandJSON",
+	New:    NewExpandJSON,
+	Config: &ExpandJSONConfig{},
+	Help:   "Explodes json objects to other fields.",
+}
+
+type ExpandJSONConfig struct {
+	Fields          map[string]string `help:"<JMESPath -> record field> map, the rest will be ignored"`
+	Source          string            `help:"record field that contains the json"`
```

Comment:
This field is required for the filter to function properly. You inform baker (and also the user when they prints the filter help) by adding the required struct tag, like so:
```suggestion
	Source          string            `help:"record field that contains the json" required:"true"`
```